### ISSUE TITLE
Using mouse to drag scroll bar in the dropdown list under edit graph -> metrics fails

### ISSUE THREAD

**Issue by bawnzai** [grafana/grafana#11744]
Read before posting: 

- Questions should be posted to https://community.grafana.com. Please search there and here on GitHub for similar issues before creating a new issue. 
- Checkout FAQ: https://community.grafana.com/c/howto/faq
- Checkout How to troubleshoot metric query issues: https://community.grafana.com/t/how-to-troubleshoot-metric-query-issues/50

Please include this information:
### What Grafana version are you using?
5.1
### What datasource are you using?
graphite
### What OS are you running grafana on?

debian jessie
### What did you do?
Edit graph -> metrics -> click a metric, get the dropdown menu for all the results
Mouse scroll wheel works, but trying to drag scroll bar immediately closes it

### What was the expected result?
Scroll through all the values in dropdown menu

### What happened instead?
Instantly closes the dropdown menu

**Comment by dehrax** [comment]
In https://github.com/grafana/grafana/blob/master/public/app/core/components/form_dropdown/form_dropdown.ts

It is possible to register an eventlistener
```javascript
typeahead.$menu.click('mousedown', (e) => {
  console.log(e.target);
});
```
Where `e.target` will be an `a`-element if one of the list items are clicked. It will be the dropdown if the scrollbar is clicked. 

I am still trying to figure out how to override the `blur` event when this is the case.

**Comment by torkelo** [comment]
I doubt this has ever worked in Grafana, so not sure it's a high prio issue :)

**Comment by minnesotags** [comment]
This bug also affects 5.2.3, firefox on debian-jessie, postgres, and the data source is InfluxDB.

If you won't fix the bug, is there an alternate way to scroll down long lists of metrics?

**Comment by marefr** [comment]
@minnesotags by typing/searching and/or scroll wheel

**Comment by oytpa** [comment]
Hi, Firefox is not permitted in our org. We use Chrome and this issue still exists. Using Grafana v5.3.4 and the SimpleJSON plugin. It is quite difficult to know beforehand what the exact words are to start typing and it is taking way too long for us to scroll using the mouse wheel (over 1000s of metrics being sent). It would be really helpful if the team could fix the UX bug.

**Comment by bawnzai** [comment]
Resolved in 6.0
